pubspec.yaml

```diff
@@ -1,8 +1,7 @@
-name: flutter_platform_network
-description: A new Flutter package project.
-version: 0.0.1
-author:
-homepage:
+name: dash_kit_network
+description: The dash component that provides basic network functionality includes token refresh.
```

Comment:
The dash component that provides basic network functionality includes token refresh. =>

The DashKit component that provides basic network functionality includes a token refreshing.

Reply:
Fixed